staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go

```diff
@@ -74,6 +74,8 @@ var (
 
 		# Diff file read from stdin
 		cat service.yaml | kubectl diff -f -`))
+
+	warningChangesOnDeletingResource = "Warning: Detected changes to resource %[1]s which is currently being deleted.\n"
```

Comment:
It makes me a little sad that we're duplicating this string, is there a good place to share it?